### DISCUSSION TITLE
STAR v2.7.10a compatible with STAR-Fusion v1.11.1?

### DISCUSSION THREAD

**Discussion by min-codes** [alexdobin/STAR#1684] [Q&A] [answered]
Hi there,

May I ask if the latest STAR (v2.7.10a) is compatible/ recommended to use with the latest STAR-Fusion (v1.11.1)? Would using these cause any significant issue if it's not listed in the compatibility matrix below? 

https://github.com/STAR-Fusion/STAR-Fusion/wiki/STAR-Fusion-release-and-CTAT-Genome-Lib-Compatibility-Matrix

Many thanks,
Min

**Comment by alexdobin** [comment, marked as answer]
Hi Min,

it should be compatible, since there were no changes in the chimeric algorithm.

**Reply by min-codes** [reply]
Sorry for the late reply! Just wanted to update here that they work fine together with the latest CTAT_Genome_lib :)
